src/persist/benches/benches.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use persist::workload::DataGenerator;
+
+mod end_to_end;
+mod snapshot;
+mod writer;
+
+pub fn bench_persist(c: &mut Criterion) {
+    let data = DataGenerator::default();
+
+    end_to_end::bench_replay(&data, &mut c.benchmark_group("end_to_end"));
```

Comment:
very very minor nit: should we follow the rough convention in the rest of the function and name this "end_to_end/replay"?

Reply:
yeah, good call